ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js

```diff
@@ -25,15 +25,15 @@ const CustomInputField = (props) => {
   return (
     <div key={props.id} style={style}>
       <TextField
-        InputLabelProps={{ style : { pointerEvents : "auto", padding : "3px" } }}
+        InputLabelProps={{ style : { pointerEvents : "auto", padding : "2px" } }}
```

Comment:
@Sonichigo thanks for this. Will you confirm equidistant spacing for the "?" and the placeholder text? Both are 2px from either left or right edge, is that right?

Reply:
Right! both are 2px

Reply:
Excellent. 😄